deploy/deploy/crane-agent/rbac.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crane-agent
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods/status
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
+  - apiGroups:
+      - ""
+    resources:
+      - nodeqosensurancepolicies.ensurance.crane.io
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - patch
+  - apiGroups:
+      - ""
+    resources:
+      - podqosensurancepolicies.ensurance.crane.io
```

Comment:
Can we design a more general thing replacing podqosensurancepolicies & nodeqosensurancepolicies?